gke/cron_testing_job.yaml.tpl

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cron-testing
+  namespace: website
+spec:
+  schedule: "0 */4 * * *"
```

Comment:
Add a comment on the actual schedule here, is this every 4 hours?

Reply:
yes, every 4 hours & added a comment